umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions-pwsh.md

```diff
@@ -0,0 +1,145 @@
+---
+description: >-
+  This section provides a step-by-step guide to setting up a CI/CD pipeline in
+  GitHub Actions using the provided sample Powershell scripts.
+---
+
+# GitHub Actions with Powershell scripts
+
+Before setting up the pipeline in GitHub, make sure that the steps in the [Configuring a CI/CD pipeline](./) are done.
+
+You will need to define your pipeline in YAML, and find a way to interact with the Umbraco Cloud Api.
+
+
+{% hint style="info" %}
+We have created a sample pipeline for GitHub. It includes YAML-files and custom PowerShell scripts to interact with the Umbraco Cloud API.
```

Comment:
okay I added too many comments about this but the spelling of API, Api, api is not consistent
also Powershell and PowerShell
also Devops and DevOps 
I think they should be uniform

Reply:
API not Api 
Powershell not PowerShell
Azure DevOps not Azure Devops
GitHub not Github
when they are used in sentences